pkg/server/image_pull.go

```diff
@@ -214,7 +264,8 @@ func (c *criContainerdService) pullImage(ctx context.Context, ref string) (
 	}
 
 	// Unpack the image layers into snapshots.
-	if _, err = c.rootfsUnpacker.Unpack(ctx, manifest.Layers); err != nil {
+	rootfsUnpacker := rootfsservice.NewUnpackerFromClient(c.rootfsService)
```

Comment:
Because in following PRs, we'll start to use rootfsService more, instead of only the `Unpacker`.

So I change the `rootfsService` to be a field for the big object, and new a unpacker when we need it. The unpacker is only needed in this function.